examples/aobench/benches/isec_sphere.rs

```diff
@@ -152,7 +148,7 @@ fn miss_vector(c: &mut Criterion) {
         "vector",
         Benchmark::new("isec_sphere_miss", move |b| {
             b.iter(|| {
-                let mut isect = IsectxN::new();
+                let mut isect = IsectxN::default();
```

Comment:
Damn it seems that the example benchmarks are never built on CI. We should build them without running them.